src/map/map.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { Controller, Get, Param, Query, HttpStatus } from '@nestjs/common';
+import { MapService } from './map.service';
+import { EachRecipeResponseDto, ReadMapDto } from './dto/read-map.dto';
+import { wrapSuccess } from '../utils/success';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+import { ResponseCallbackDto } from 'src/auth/dto/response-callback.dto';
+
+@Controller('maps')
+@ApiTags('maps')
+export class MapController {
+  constructor(private readonly mapService: MapService) {}
+
+  @Get('/:id?')
```

Comment:
```suggestion
  @Get(':id')
```

@Query() 데코레이터 사용하면 '?' 는 라우팅에 명시하지 않아도 되지 않나요?
그리고 앞쪽 '/' 는 빼고 쓰는거로 알고 있습니다.

[nest controller](https://docs.nestjs.com/controllers#request-payloads)

Reply:
아그리고 저도 이제 알았는데 '/' 맨 앞에는 안붙여도 되나봐요 (?)